kliff/trainer/torch_trainer_utils/dataloaders.py

```diff
@@ -82,33 +185,22 @@ def collate(self, batch: Any) -> dict:
         """
         # get fingerprint and consistent properties
         config_0, property_dict_0 = batch[0]
-        device = config_0.device
-        ptr = torch.tensor([0], dtype=torch.int64, device=device)
+        ptr = np.array([0], dtype=np.intc)
```

Comment:
I think the collated data will be provided as the input to a torch NN model. Any reason why use `np.array` but `torch.tensor` for all variables?

Reply:
Update: NVM, now I understand that the batched data are passed to the descriptor in `def _descriptor_eval_batch(self, batch) -> torch.Tensor`, which requires numpy array.